src/app/services/offre-de-stage/offre-de-stage.service.ts

```diff
@@ -27,9 +27,9 @@ export class OffreDeStageService {
   }
 
   getOffreDeStage(
-    _id: string
+    _id?: string
   ): Observable<{ success: boolean; data?: OffreDeStage }> {
-    return this.http.get<{ success: boolean; data?: OffreDeStage }>(
+    return this.http.get<{ success: boolean; data: OffreDeStage }>(
```

Comment:
Si `success=false`, `data` n'existera pas.
```suggestion
    return this.http.get<{ success: boolean; data?: OffreDeStage }>(
```